pkg/agent/util/net_windows.go

```diff
@@ -524,5 +508,74 @@ func HostInterfaceExists(ifaceName string) bool {
 func SetInterfaceMTU(ifaceName string, mtu int) error {
 	cmd := fmt.Sprintf("Set-NetIPInterface -IncludeAllCompartments -InterfaceAlias \"%s\" -NlMtuBytes %d",
 		ifaceName, mtu)
-	return InvokePSCommand(cmd)
+	_, err := ps.RunCommand(cmd)
+	return err
+}
+
+func NewNetRoute(route *Route) error {
+	cmd := fmt.Sprintf("New-NetRoute -InterfaceIndex %v -DestinationPrefix %v -NextHop %v -RouteMetric %d -Verbose",
+		route.LinkIndex, route.DestinationSubnet.String(), route.GatewayAddress.String(), route.RouteMetric)
+	_, err := ps.RunCommand(cmd)
+	return err
+}
+
+func RemoveNetRoute(route *Route) error {
+	cmd := fmt.Sprintf("Remove-NetRoute -InterfaceIndex %v -DestinationPrefix %v -NextHop  %v -Verbose -Confirm:$false",
+		route.LinkIndex, route.DestinationSubnet.String(), route.GatewayAddress.String())
+	_, err := ps.RunCommand(cmd)
+	return err
+}
+
+func GetNetRoutes(linkIndex int, dstSubnet *net.IPNet) ([]Route, error) {
+	cmd := fmt.Sprintf("Get-NetRoute -InterfaceIndex %d -DestinationPrefix %s -erroraction Ignore | Format-Table -HideTableHeaders",
+		linkIndex, dstSubnet.String())
+	return getNetRoutes(cmd)
+}
+
+func GetNetRoutesAll() ([]Route, error) {
+	cmd := "Get-NetRoute -ErrorAction Ignore | Format-Table -HideTableHeaders"
+	return getNetRoutes(cmd)
+}
+
+func getNetRoutes(cmd string) ([]Route, error) {
+	routesStr, _ := ps.RunCommand(cmd)
+	routes, err := parseRoutes(routesStr)
+	if err != nil {
+		return nil, err
+	}
+	return routes, nil
+}
+
+func parseRoutes(routesStr string) ([]Route, error) {
+	scanner := bufio.NewScanner(strings.NewReader(routesStr))
+	var routes []Route
+	for scanner.Scan() {
+		items := strings.Fields(scanner.Text())
+		if len(items) < 4 {
```

Comment:
I chose 4 because we only need 4. However, now I don't think it is a good idea. The format of `Get-NetRoute` is 6 elements in a row and we should follow it. Updated.